packages/bot-web-ui/src/pages/dashboard/announcements/announcement-components.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { LabelPairedCircleInfoCaptionBoldIcon } from '@deriv/quill-icons';
```

Comment:
We need to add `@deriv/quill-icons` in bot-web-ui/package.json

@maryia-matskevich-deriv
 
<img width="1705" alt="Screenshot 2024-08-13 at 12 13 19 PM" src="https://github.com/user-attachments/assets/a592cade-ad85-46e5-b851-9db8034d676f">